src/backends/fixerio/index.js

```diff
@@ -60,26 +51,15 @@ class FixerIoBackend {
       })
       apiData = result.body
     }
-
+    const pairs = config.get('tradingPairs')
+    this.pairs = pairs.map((p) => [p[0].slice(0, 3),
```

Comment:
It's confusing to assign `this.pairs` to a thing that is different than `pairs`. Maybe this should be called `currencyCodePairs`?